src/Elastic.Apm/Model/Span.cs

```diff
@@ -17,6 +17,7 @@ internal class Span : ISpan
 		private readonly Transaction _enclosingTransaction;
 		private readonly IApmLogger _logger;
 		private readonly IPayloadSender _payloadSender;
+		private bool _dbgIsEnded;
```

Comment:
Sure about the `dbg` prefix? This variable influences the behaviour of the `End()` method.

Reply:
Yes, you're right - I've removed `dbg` prefix.